tools/owncloud-unlockadmin.sh

```diff
@@ -8,15 +8,15 @@
 
 source /etc/mailinabox.conf # load global vars
 
-ADMIN=$(./mail.py user admins | head -n 1)
```

Comment:
That was actually just redirecting to management/cli.py, hence fixing the path and removing that file. All works fine.